Documentation/Internal/ContainerLogV2-Linux.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MonitoringManagement version="1.0" namespace="<NamespaceForLinuxContainers>" eventVersion="1" timestamp="2016-01-20T00:00:00.000">
+  <Accounts>
+    <Account moniker="<GenevaLogsAccountMoniker>" isDefault="true" />
+  </Accounts>
+  <Management eventVolume="Large" defaultRetentionInDays="90">
+    <Identity>
+      <IdentityComponent name="Tenant">TENANT_ID</IdentityComponent>
+      <IdentityComponent name="Role">ROLE_ID</IdentityComponent>
+      <IdentityComponent name="RoleInstance">ROLEINSTANCE_ID</IdentityComponent>
+    </Identity>
+    <AgentResourceUsage diskQuotaInMB="50000" />
+  </Management>
+  <Schemas>
+   <Schema name="ContainerLogV2Schema">
+      <Column name="ContainerId" type="str" mdstype="mt:wstr" />
+      <Column name="LogMessage" type="str" mdstype="mt:wstr" />
+      <Column name="LogSource" type="str" mdstype="mt:wstr" />
+      <Column name="TimeGenerated" type="str" mdstype="mt:wstr" />
+      <Column name="PodName" type="str" mdstype="mt:wstr" />
+      <Column name="PodNamespace" type="str" mdstype="mt:wstr" />
+      <Column name="Computer" type="str" mdstype="mt:wstr" />
+      <Column name="ContainerName" type="str" mdstype="mt:wstr" />
+      <Column name="AzureResourceId" type="str" mdstype="mt:wstr" />
```

Comment:
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/66e9ff82-b532-44ee-803c-88b67ee1cdc6)
are those two schemas have different column on the AzureResourceId?

Reply:
> ![image](https://user-images.githubusercontent.com/25573844/253696170-66e9ff82-b532-44ee-803c-88b67ee1cdc6.png) are those two schemas have different column on the AzureResourceId?

mdsd.xml for the Log analytics and the config xmls in this PR for geneva.